desktop/libs/notebook/src/notebook/api.py

```diff
@@ -571,7 +571,17 @@ def _historify(notebook, user):
 
 def _get_statement(notebook):
   if notebook['snippets'] and len(notebook['snippets']) > 0:
-    return Notebook.statement_with_variables(notebook['snippets'][0])
+    snippet = notebook['snippets'][0]
+    try:
+      if snippet.get('executor', {}).get('executables', []):  # With Connectors/Editor 2
+        executable = snippet['executor']['executables'][0]
```

Comment:
Is it always the first?

Reply:
"Could be", and it is funky right now as we can have 10s of executables. This is more of a temp solution until the content we send/pass around is trimmed to be clean. Adding a jira